examples/plot_vertical.py

```diff
@@ -25,3 +25,16 @@
 plot(example, orientation='vertical', show_counts='%d', show_percentages=True)
 plt.suptitle('With counts and percentages shown')
 plt.show()
+
+#########################################################################
+# An UpSetplot with additional plots on vertical
+# and tuning some visual parameters
+example = generate_counts(extra_columns=2)
```

Comment:
I think using generate_samples here makes more sense? But maybe 10k samples is a lot for three swam plots.